app/assets/javascripts/comments.js

```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
+// All this logic will automatically be available in application.js.
```

Comment:
I would refer to you to — https://martinfowler.com/bliki/BeckDesignRules.html

My interpretation is:

1. Does it reveal intention? Maybe sometimes, but in these cases not — the files with the actual functionality in them reveal all the useful intention here, and these files communicate nothing about what we might use them for. At the very most they communicate 'a developer added these intending to do... something, that I have no way of knowing'.
2. Since it doesn't, defer to 'fewest elements' and remove them.

It's also worth remembering that adding them back in should we decide we need them is as easy as creating a file! So not a big cost.